tutorials/7-cosmjs/5-create-custom.md

```diff
@@ -31,120 +31,38 @@ This section assumes that you have a working Cosmos blockchain with its own modu
 
 ## Compiling the Protobuf objects and clients
 
-You can choose which library you use to compile your Protobuf objects into TypeScript or JavaScript. Reproducing [what Stargate](https://github.com/cosmos/cosmjs/blob/main/packages/stargate/CUSTOM_PROTOBUF_CODECS.md) or [`cosmjs-types`](https://github.com/confio/cosmjs-types/blob/main/scripts/codegen.js) do is a good choice.
+You can choose which library you use to compile your Protobuf objects into TypeScript or JavaScript. Reproducing what [cosmjs-types](https://github.com/confio/cosmjs-types/blob/main/scripts/codegen.js) or [Stargate](https://github.com/cosmos/cosmjs/blob/main/packages/stargate/CUSTOM_PROTOBUF_CODECS.md) do is a good choice.
 
 ### Preparation
 
 This exercise assumes that:
 
-1. Your Protobuf definition files are in `./proto/myChain`.
-2. You want to compile them into TypeScript in `./client/src/types/generated`.
+1. The library you're creating is in 'myLib'.
+2. Your Protobuf definition files are in `./proto/`.
+3. You want to transpile them into TypeScript in `./src/codegen/`.
 
-Install `protoc` on your computer and its Typescript plugin in your project, possibly with the help of a Dockerfile:
-
-<CodeGroup>
-
-<CodeGroupItem title="Local">
+Install `telescope` on your computer globally:
 
 ```sh
-$ mkdir -p /usr/lib/protoc
-$ cd /usr/lib/protoc
-$ curl -L https://github.com/protocolbuffers/protobuf/releases/download/v21.7/protoc-21.7-linux-x86_64.zip -o protoc.zip
-$ unzip -o protoc.zip
-$ rm protoc.zip
-$ ln -s /usr/lib/protoc/bin/protoc /usr/local/bin/protoc
-$ npm install ts-proto@1.121.6 --save-dev
+npm install -g @cosmology/telescope
 ```
 
-Adjust to your preferred version, operating system, and CPU platform. For instance, on an Apple M1 you would use `protoc-21.7-osx-aarch_64.zip`.
-
-</CodeGroupItem>
-
-<CodeGroupItem title="Dockerfile">
```

Comment:
Please adjust so that we keep both Local and Dockerfile.

Reply:
Hi, Thank you very much for the review! 
Please excuse me that I don't understand how can I keep the Dockerfile part for this. I think with --save-dev and --save-exact as you suggested, telescope can be installed locally in the package, so we don't have to use docker in this case. Did you mean that I should also keep protoc part? I think that will make this tutorial more complicated... 
Other suggestions are great! I'll fix them in the PR in developer-portal

Reply:
If we don't need the `protoc` executable anymore because Telescope is taking care of this, and it is a single `npm` package, then perhaps we don't need Docker.